core/lib/Cardano/Address/Style/Shelley.hs

```diff
@@ -522,65 +524,75 @@ parseAddressInfoShelley AddressParts{..} = case addrType of
             { infoStakeReference = Just ByValue
             , infoSpendingKeyHash = Just addrHash1
             , infoStakeKeyHash = Just addrHash2
+            , infoAddressType = "0000"
```

Comment:
Wouldn't it make more sense to show the base-10 value of the type? i.e. here type `0`

Reply:
yes, but then icarus and byron address will not be differentiable only looking at this field..both assume "1000"

Reply:
But how would showing the bit sequence make it any better for Icarus and Byron :no_mouth: ?

Reply:
agree, fix applied https://github.com/input-output-hk/cardano-addresses/pull/157/commits/ae2a6ff87846d615f801ac4e7430872f75563d97